README.md

```diff
@@ -41,6 +41,10 @@ production, unless you ensure access to the NREPL port is restricted.
 
 This will start up with non-persistent in-memory storage only.
 
+**Remark**:
+if you use java 1.7, you might need to add `-XX:MaxPermSize=256m`
+to java options.
```

Comment:
Yes while doing this PR I was annoyed by the double warning each time I launch a lein run. So I moved the `-XX:MaxPermSize=256` in the travis-ci as an environment variable. So the warning is only visible during travis CI tests with java-1.8. (this is the first commit of this PR)